src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Program.cs

```diff
@@ -5,6 +5,9 @@
 #if (WindowsAuth)
 using Microsoft.AspNetCore.Authentication.Negotiate;
 #endif
+#if (EnableOpenAPI)
+using Microsoft.AspNetCore.OpenApi;
```

Comment:
Do we normally add `using`s without adding code that uses the `using`?

Reply:
Good point! I added this as a nudge to the existence of the package and for compat with the other `Program.cs` but it's probably find to omit it for a first round.

EDIT: I also just realized this is the modern day controllers template, not the minimal template. Removed.